app/posts/ecf-v2/2024-07-15-workshop-process.md

```diff
@@ -0,0 +1,69 @@
+---
+title: The process behind our problem prioritisation and ideation workshops
+description: "Our journey of prioritising problems and developing high level solutions to take forward"
+date: 2024-07-15
+---
+
+Over the last month our team ran several workshops with the aim of consolidating our top problems across our 4 user groups and prioritising them followed by ideating on solutions on how to address our biggest problems. This consisted of 4-5 problems workshops and 1-2 ideation workshops.
+
```

Comment:
```suggestion
Over the last month our team ran several workshops.

We aimed to:
* consolidate, prioritise and better understand the top problems across the digital services that facilitate the early career framework (ECF) policy reforms
* ideate on solutions for these problems

This consisted of five problem workshops and two ideation workshops.

```

Reply:
Some suggestions for simplications as the first sentence was a bit wordy. :) 

i don't think saying user groups without defining is too helpful, so also suggested the wording Mili gave to me for the remit of our team

Reply:
You define the user groups later too, so think its OK to simplify here